src/components/Legend/FlexRow.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+export const FlexRow = styled.div`
```

Comment:
Same here. These components have very generic names but very precise stylings. Please rather put all of your StyledComponents above the components that use it. Unless they are used by many components in which case it should be more explicit what it is made for.